bumble/host.py

```diff
@@ -729,17 +914,31 @@ def on_hci_command_complete_event(self, event):
     def on_hci_command_status_event(self, event):
         return self.on_command_processed(event)
 
-    def on_hci_number_of_completed_packets_event(self, event):
+    def on_hci_number_of_completed_packets_event(
+        self, event: hci.HCI_Number_Of_Completed_Packets_Event
+    ) -> None:
         for connection_handle, num_completed_packets in zip(
             event.connection_handles, event.num_completed_packets
         ):
             if connection := self.connections.get(connection_handle):
-                connection.acl_packet_queue.on_packets_completed(num_completed_packets)
-            elif connection_handle not in itertools.chain(
-                self.cis_links.keys(),
-                self.sco_links.keys(),
-                itertools.chain.from_iterable(self.bigs.values()),
-            ):
+                connection.acl_packet_queue.on_packets_completed(
```

Comment:
May use the new method?
```suggestion
                if queue := self.get_data_packet_queue(connection_handle):
                  queue.on_packets_completed(num_completed_packets, connection_handle)
```

Reply:
done